tests_e2e/test_e2e_tests.py

```diff
@@ -0,0 +1,89 @@
+import os
+from threading import Thread
+import pytest
+import uuid
+from selenium import webdriver
+import app
+import board_api as board_api
+
+@pytest.fixture(scope='module')
+def test_app():    
+    board_name = f'Corndell_tests_{uuid.uuid4()}'
```

Comment:
Unless you are running with e.g. VSCode settings to automatically load the .env file, you'll need to add `load_dotenv(find_dotenv('.env'), override=True)` here for the tests to pass. Then you should be able to run `pytest tests_e2e` from command line as well.